tutorials/43_Building_a_Tool_Calling_Agent.ipynb

```diff
@@ -268,7 +263,11 @@
     "from haystack.components.agents import Agent\n",
     "from haystack.components.generators.chat import OpenAIChatGenerator\n",
     "\n",
-    "search_component = SuperComponent(pipeline=search_pipeline)\n",
+    "search_component = SuperComponent(\n",
+    "    pipeline=search_pipeline,\n",
+    "    input_mapping={\"query\": [\"search.query\"]},\n",
+    "    output_mapping={\"output_adapter.output\": \"search_result\"}\n",
```

Comment:
@bilgeyucel I added these here to show how to use these in a SuperComponent context as well as to make the parameter specification of the ComponentTool only contain query instead of all possible inputs the SuperComponent could take.